ffig/generators/python.py

```diff
@@ -0,0 +1,24 @@
+# Generator module for Python 2 and 3.
+
+import generators
+import os
+
+def generator(module_name, binding, api_classes, env, args, output_dir):
+    module_dir = os.path.join(output_dir, module_name)
```

Comment:
(Including `generators.generate`, and `generators.GeneratorContext.generate`; if the list of files isn't propagated right back to `FFIG.py`, it's as good as useless.